src/main/java/core/basesyntax/strategy/SupplyOperationHandler.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class SupplyOperationHandler implements OperationHandler {
+
+    @Override
+    public void updateFruitStorage(FruitTransaction fruitTransaction,
+                                   FruitStorageDao fruitStorageDao) {
+        fruitStorageDao.increaseQuantity(fruitTransaction);
+    }
```

Comment:
The `SupplyOperationHandler` class correctly implements the `OperationHandler` interface. However, according to the checklist, it is important to ensure that all logic related to the operation is encapsulated within this handler. Make sure that any validation or additional logic that is specific to the 'supply' operation is also handled here, rather than just delegating the call to `increaseQuantity`.